src/server.js

```diff
@@ -115,7 +114,9 @@ const initialize = async (port = Config.API_PORT, isApi = true) => {
   await migrate(isApi)
   const server = await createServer(port, isApi)
   server.plugins.openapi.setHost(server.info.host + ':' + server.info.port)
-  Logger.info(`Server running on ${server.info.host}:${server.info.port}`)
+  console.log('Server running on', server.info)
```

Comment:
why console.log? we should not be using that, use logger?

Reply:
Yeah this was just me figuring out how to fix the tests as `Logger` didn't exist